tool/planet/start.go

```diff
@@ -197,6 +197,18 @@ func start(config *Config, monitorc chan<- bool) (*runtimeContext, error) {
 			Val:  strings.Join(upstreamNameservers, ","),
 		})
 
+	// Add address of local nameserver to environment.
```

Comment:
Nit: comment should answer why. 
```
// Add address of local nameserver to environment so that processes launched within planet such as planet-agent can locate the local nameserver for testing.
```